modules/custom-tuning-specification.adoc

```diff
@@ -0,0 +1,136 @@
+
+Multiple custom tuning specifications can co-exist as multiple CRs in the
+operator's namespace. The existence of new CRs or the deletion of old CRs is
+detected by the Operator. All existing custom tuning specifications are merged
+and appropriate objects for the containerized tuned daemons are updated.
+
+*profile: data*
+
+The `profile:` section lists tuned profiles and their names.
+
+----
+profile:
+- name: tuned_profile_1
+  data: |
+    # Tuned profile specification
+    [main]
+    summary=Description of tuned_profile_1 profile
+
+    [sysctl]
+    net.ipv4.ip_forward=1
+    # ... other sysctl's or other tuned daemon plugins supported by the containerized tuned
+
+# ...
+
+- name: tuned_profile_n
+  data: |
+    # Tuned profile specification
+    [main]
+    summary=Description of tuned_profile_n profile
+
+    # tuned_profile_n profile settings
+----
+
+*Recommended profiles*
+
+The `profile:`selection logic is defined by the recommend section of the CR:
+
+----
+recommend:
+- match:                              # optional; if omitted, profile match is assumed unless a profile with a higher matches first
+  <match>                             # an optional array
+  priority: <priority>                # profile ordering priority, lower numbers mean higher priority (0 is the highest priority)
+  profile: <tuned_profile_name>       # e.g. tuned_profile_1
+
+# ...
+
+- match:
+  <match>
+  priority: <priority>
+  profile: <tuned_profile_name>       # e.g. tuned_profile_n
+----
+
+If `<match>` is omitted, a profile match (for example, `true`) is assumed.
+
+`<match>` is an optional array recursively defined as follows:
+
+----
+- label: <label_name>     # node or pod label name
+  value: <label_value>    # optional node or pod label value; if omitted, the presence of <label_name> is enough to match
+  type: <label_type>      # optional node or pod type ("node" or "pod"); if omitted, "node" is assumed
+  <match>                 # an optional <match> array
+----
+
+If `<match>` is not omitted, all nested `<match>` sections must also evaluate to
+`true`. Otherwise, `false` is assumed and the profile with the respective
+`<match>` section will not be applied or recommended. Therefore, the nesting
+(child `<match>` sections) works as logical and Operator. Conversely, if any
+item of the `<match>` array matches, the entire `<match>` array evaluates to
+`true`. Therefore, the array acts as logical or Operator.
+
+.Example
+
+----
+- match:
+  - label: tuned.openshift.io/elasticsearch
+    match:
+    - label: node-role.kubernetes.io/master
+    - label: node-role.kubernetes.io/infra
+    type: pod
+  priority: 10
+  profile: openshift-control-plane-es
+- match:
+  - label: node-role.kubernetes.io/master
+  - label: node-role.kubernetes.io/infra
+  priority: 20
+  profile: openshift-control-plane
+- priority: 30
+  profile: openshift-node
+----
+
+The CR above is translated for the containerized tuned daemon into its
```

Comment:
s/CR/CRD ??

Reply:
Why? https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
You're creating a CRD's instance, a CR.

Reply:
I will defer to other's opinion. Perhaps my understanding isn't correct. I was looking elsewhere in our docs (1) where the file is called a CRD that creates the custom object. 
(1) https://docs.openshift.com/container-platform/3.11/admin_guide/custom_resource_definitions.html#creating-crd_admin-guide-custom-resources

Reply:
Unless I'm wrong:
custom resource definition (CRD) object = custom resource (CR)

The document states "the CR above is translated ...".  CRD is definitely not translated here, it is the CRD object that is translated, i.e. CR.  Therefore unless we cannot talk about CRs, we need to talk about CRD objects here.